corporate/lib/stripe.py

```diff
@@ -3712,6 +3712,9 @@ def do_change_remote_server_plan_type(remote_server: RemoteZulipServer, plan_typ
 
 @transaction.atomic
 def do_deactivate_remote_server(remote_server: RemoteZulipServer) -> None:
+    # TODO: This should also ensure that the server doesn't have an active plan,
+    # and deactivate it otherwise. (Like do_deactivate_realm does.)
+
```

Comment:
This isn't needed for this functionality right now, but adding the TODO  since we'll to do this if we want to integrate the deactivation functionality into the billing system in a proper way.

Reply:
Yeah we should probably fix that in the next day, even if we merge this without waiting for that.